Vocabularies/Cage.js

```diff
@@ -0,0 +1,36 @@
+function cageVocabulary() {

```

Comment:
ChastityCage Vocab exists?

Reply:
I probably kept  it to resolve a dangling reference